### DISCUSSION TITLE
Power BI Project Live Connected report

### DISCUSSION THREAD

**Discussion by suketududhat** [TabularEditor/TabularEditor3#1290] [🙋♀️TE3 Q&A] [answered]
Is there any possibility of this capability in TE3? If not, is there a way to fake the model by including the bim in the project folder?

All our reports connect to central semantic models via live connection so TE3 is unable to open the pbip file. We want to leverage the new preview format for reports and easily make changes in TE3 but for now we are limited to just the models.

**Comment by otykier** [comment, marked as answer]
@suketududhat Tabular Editor is a tool for editing semantic models. Live connected reports does not contain any semantic models, and as such, there is nothing for Tabular Editor to edit.

We do not have any plans to support editing the report/visual metadata through Tabular Editor. This is mostly because Microsoft has not provided a programmatic way to access the report/visual metadata (i.e. we need an equivalent of the [AMO/TOM client library](https://learn.microsoft.com/en-us/analysis-services/amo/developing-with-analysis-management-objects-amo?view=asallproducts-allversions), but for the report metadata).

**Reply by suketududhat** [reply]
Saving to PBIB creates report metadata files and one of them is a report.json that should in theory allow an external tool to edit, but it sounds like it would still require a report client library to programmatically make changes?

Editing report.json in VS Code by replacing column names of a visual with some other column in semantic model does work so hopefully Microsoft is working on allowing other tools to make changes.

**Reply by KornAlexander** [reply]
@otykier what about the "report specific measures" in live connected reports? 
Background of question: I would love to edit the translations of those "report specific measures" as well.

**Reply by otykier** [reply]
To my knowledge, report specific objects does not support translations. You’ll have to use one of the workarounds described here: https://pbi-guy.com/2022/01/31/multilingual-reports-in-power-bi/
